### PR TITLE
Changelog for 1.10.2

### PR DESCRIPTION
**Pull request by akshayhavile**

### CODE REVIEW

**Comment by akshayhavile** [comment]
skip jira-id-check

File: CHANGELOG.md [hunk 1]
```diff
@@ -364,4 +364,14 @@ All notable changes to this project will be documented in this file. The format
...
 ### Known Issues
- - AKO doesn’t support Service of type LoadBalancer definitions with mixed protocols.
\ No newline at end of file
+ - AKO doesn’t support Service of type LoadBalancer definitions with mixed protocols.
+
+## AKO-1.10.2
+
+### Added
+ - AKO now claims support for K8 1.27.
+ - AKO chooses correct network reference for VIP subnet and Pool placement when there are duplicate networks, with same name, present on Avi Controller.
```
Comment (swathinsankaran) [inline CHANGELOG.md:373]: Do you think we should include this under fixed since it is a bug fix?

**Comment by akshayhavile** [comment]
build ako

**Review by swathinsankaran — COMMENTED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (swathinsankaran) [inline CHANGELOG.md:373]: Do you think we should include this under fixed since it is a bug fix?

Comment (akshayhavile) [reply CHANGELOG.md:373]: `Duplicate network being shown` is issue from NSX, VCenter and Avi controller side. AKO just added capability to detect right network from those duplicate network. I feel that if we add that as part of fixed issue, Customer will assume AKO had issue in choosing correct network.

**Review by swathinsankaran — APPROVED** [review]

**Pull request merged**